### PR TITLE
Add Fenn to Rookie list

### PR DESCRIPTION
**Pull request by fennsaji**
# Description

Welcome! We are excited to have you on board. This is your first formal step in making it official that you are a part of the Polkadot Heroes program as either a hero or a rookie. Please fill out your profile details in this pull request. Remember to place your profile in alphabetical order in the list, add your profile image to the `./images` subfolder of either the `Heroes/` or `Rookies/` folder, and fill out all the informational fields in your card. You can copy and paste someone else's card and replace their information with your own.

Make sure to preview your additions in the Heroes/README.md or the Rookies/README.md file to confirm it looks as expected.

# Checklist:

- [x] My name is alphabetically ordered by first name in the README
- [x] My profile picture is added to the `/images` folder of either the `Heroes/` or the `Rookies/` directory
- [x] All the fields are filled out for my profile (see the other profiles for examples)

### CODE REVIEW

File: Rookies/README.md [hunk 1]
```diff
@@ -19,6 +19,17 @@ You can read the [Heroes Review Criteria and Scoring Matrix Guidelines](../Heroe
 - Personal Site: [https://www.parity.io/](https://www.parity.io/)
 - Blog: [https://www.parity.io/blog/](https://www.parity.io/blog/)
 
+
+## Fenn Ignatius Saji
```
Comment (hummusonrails) [inline Rookies/README.md:23]: If you could just change this to a H3: `###` that would be great:

```suggestion
### Fenn Ignatius Saji
```

**Comment by hummusonrails** [comment]
Hey @fennsaji!

Welcome to the Rookies and congrats on being the first one to complete this step! 🎉 🎉 

I left just one small comment, and after that change, we're good to go! 🚀 

- Ben

File: Rookies/README.md (hunk 1 above)
Comment (fennsaji) [reply Rookies/README.md]: > If you could just change this to a H3: `###` that would be great:

Updated

**Comment by fennsaji** [comment]
Thanks, @hummusonrails

**Comment by hummusonrails** [comment]
@fennsaji - Can you do a small favor and rebase your fork to incorporate the latest changes to the file and then push up again to this PR?

**Comment by fennsaji** [comment]
@hummusonrails Updated

**Pull request merged**